t/widgets/render-methods.lisp

```diff
@@ -11,6 +11,7 @@
                 #:is-html))
 (in-package weblocks-test/widgets/render-methods)
 
+(setf weblocks/html:*pretty-html* nil)
```

Comment:
Please, don't change this on the package level, because somebody can load `weblocks-test` system during their normal workflow. And the output will be changed not only for tests but also for the whole project.

Change `is-html` macro to bind `weblocks/html:*pretty-html*` to nil.